swimlane/core/adapters/report.py

```diff
@@ -53,4 +53,6 @@ def build(self, name, **kwargs):
         Returns:
             Report: Newly created local Report instance
         """
+        if (kwargs["limit"] < 0):
```

Comment:
This would be a better approach, since we need to reject the following:

* Negative numbers
* Non-integers
* Null values ("None" in Python terms)

https://github.com/swimlane/swimlane-python/commit/bd24503a2cc5df7fb981851372bc2a40318f7e5f#diff-04c4619224c035bddba8407143dff0e51523432bce22ef1404f68e83ed46b715R117